stats.go

```diff
@@ -51,7 +52,69 @@ func getInitStats() *types.Statistics {
 	stats.Falco.Add("informational", 0)
 	stats.Falco.Add("debug", 0)
 	stats.Falco.Add("unknown", 0)
-
+	stats.Slack.Add("total", 0)
+	stats.Slack.Add("error", 0)
+	stats.Slack.Add("ok", 0)
+	stats.Rocketchat.Add("total", 0)
+	stats.Rocketchat.Add("error", 0)
+	stats.Rocketchat.Add("ok", 0)
+	stats.Mattermost.Add("total", 0)
+	stats.Mattermost.Add("error", 0)
+	stats.Mattermost.Add("ok", 0)
+	stats.Teams.Add("total", 0)
+	stats.Teams.Add("error", 0)
+	stats.Teams.Add("ok", 0)
+	stats.Datadog.Add("total", 0)
+	stats.Datadog.Add("error", 0)
+	stats.Datadog.Add("ok", 0)
+	stats.Discord.Add("total", 0)
+	stats.Discord.Add("error", 0)
+	stats.Discord.Add("ok", 0)
+	stats.Alertmanager.Add("total", 0)
+	stats.Alertmanager.Add("error", 0)
+	stats.Alertmanager.Add("ok", 0)
+	stats.Elasticsearch.Add("total", 0)
+	stats.Elasticsearch.Add("error", 0)
+	stats.Elasticsearch.Add("ok", 0)
+	stats.Influxdb.Add("total", 0)
+	stats.Influxdb.Add("error", 0)
+	stats.Influxdb.Add("ok", 0)
+	stats.Loki.Add("total", 0)
+	stats.Loki.Add("error", 0)
+	stats.Loki.Add("ok", 0)
+	stats.Nats.Add("total", 0)
+	stats.Nats.Add("error", 0)
+	stats.Nats.Add("ok", 0)
+	stats.AWSLambda.Add("total", 0)
+	stats.AWSLambda.Add("error", 0)
+	stats.AWSLambda.Add("ok", 0)
+	stats.AWSSQS.Add("total", 0)
+	stats.AWSSQS.Add("error", 0)
+	stats.AWSSQS.Add("ok", 0)
+	stats.AWSSNS.Add("total", 0)
+	stats.AWSSNS.Add("error", 0)
+	stats.AWSSNS.Add("ok", 0)
+	stats.SMTP.Add("total", 0)
+	stats.SMTP.Add("error", 0)
+	stats.SMTP.Add("ok", 0)
+	stats.Opsgenie.Add("total", 0)
+	stats.Opsgenie.Add("error", 0)
+	stats.Opsgenie.Add("ok", 0)
+	stats.Statsd.Add("total", 0)
+	stats.Statsd.Add("error", 0)
+	stats.Statsd.Add("ok", 0)
+	stats.Dogstatsd.Add("total", 0)
+	stats.Dogstatsd.Add("error", 0)
+	stats.Dogstatsd.Add("ok", 0)
+	stats.Webhook.Add("total", 0)
+	stats.Webhook.Add("error", 0)
+	stats.Webhook.Add("ok", 0)
+	stats.AzureEventHub.Add("total", 0)
+	stats.AzureEventHub.Add("error", 0)
+	stats.AzureEventHub.Add("ok", 0)
+        stats.GCPPubSub.Add("total", 0)
+        stats.GCPPubSub.Add("error", 0)
+        stats.GCPPubSub.Add("ok", 0)
```

Comment:
wrong identation, and see my preivous comment, this type of declaration is now deprecated